src/components/Blogs/BlogComments/AddCommentForm.jsx

```diff
@@ -35,20 +33,34 @@ const AddCommentForm = (props) => {
 
     return (
         <AddCommentFormContainer>
-            <Form onSubmit={handleSubmit}>
+            <StyledForm onSubmit={handleSubmit}>
                 <Input
                     type="text"
                     name="comment"
-                    id={"comment"}
+                    id="comment"
                     value={comment}
                     onChange={onChange}
                     placeholder="Add your comment here..."
                 />
                 <ButtonGreen width={"100%"} type="submit">
                     Submit
                 </ButtonGreen>
-            </Form>
-            {error && <p style={{ marginTop: "10px" }}>{error}</p>}
+            </StyledForm>
+            {error && (
+                <p
+                    style={{
+                        position: "fixed",
+                        bottom: "20px",
+                        right: "20px",
+                        backgroundColor: "red",
+                        color: "white",
+                        padding: "10px",
+                        borderRadius: "5px",
+                    }}
+                >
+                    {error}
+                </p>
+            )}
```

Comment:
@Saifmagouri nope delete it and use toast.warn method from "react-toastify" package